### ISSUE TITLE
For anybody else finding their way here...

### ISSUE THREAD

**Issue by TranquilMarmot** [dvdizon/kbd75-qmk-guide#6]
This guide is great but I don't think this is all necessary anymore!

The QMK firmware docs should be looked at instead: https://docs.qmk.fm/#/

TL;DR:
- Go to https://config.qmk.fm/ and load your keyboard and change the layout to your liking
- Click the "Compile" button in the top right
- This will take a minute, but you should eventually be able to click the "Firmware" button to download a `.hex` file
- Download QMK toolbox from here: https://github.com/qmk/qmk_toolbox/releases
- Open the `.hex` file from above
- Follow the steps here to put your keyboard into bootloader mode: https://docs.qmk.fm/#/newbs_flashing
    - For me, this was holding down spacebar + b while plugging the keyboard in
- You should see some yellow text in QMK toolbox
- Click the "Flash" button
- Done 😃

**Comment by dvdizon** [comment]
Resolved via PR #7
